src/lib.rs

```diff
@@ -88,6 +132,10 @@ impl PrivateKey {
             .map(PrivateKey)
             .map_err(|_| JsValue::from_str("Invalid normal secret key"))
     }
+
+    pub fn sign(&self, message: &[u8]) -> Ed25519Signature {
+        Ed25519Signature(self.0.sign(&message.to_vec()))
```

Comment:
why do you need `to_vec` here?

Reply:
Because `Ed25519Signature` is defined as `impl_signature!(Ed25519Signature, Vec<u8>, crypto::Ed25519);` (this means the `signee_type` is `Vec<u8>`) so we need to sign a `Vec<u8>`